pkg/kubelet/status/status_manager.go

```diff
@@ -66,11 +65,26 @@ type versionedPodStatus struct {
 	status v1.PodStatus
 }
 
+// podManager abstracts access to information about pods on the Kubelet.
+type podManager interface {
```

Comment:
nit: It may make sense to document the interface here similar to the one in volume manager, i.e. something like:

```
// podManager is the subset of methods the status manager manager needs to observe the actual state of the kubelet
```

(and then the actual godoc for all the methods can be on the main pod manager interface). Otherwise, it will be duplicated between the "subset" of the interface here and the actual interface in `pod_manager.go`. As a result, I'm guessing will be tricky to be keep in sync between them.

Reply:
Agreed